packs/feats/bond-conservation.json

```diff
@@ -28,7 +28,41 @@
             "remaster": true,
             "title": "Pathfinder Player Core"
         },
-        "rules": [],
+        "rules": [
+            {
+                "itemType": "spell",
+                "key": "ItemAlteration",
+                "mode": "add",
+                "predicate": [
+                    "feature:arcane-bond",
+                    "spellshape:bond-conservation"
+                ],
+                "property": "description",
+                "value": [
+                    {
+                        "text": "PF2E.SpecificRule.Spellshape.BondConservation"
+                    }
+                ]
+            },
+            {
+                "key": "RollOption",
+                "label": "PF2E.TraitSpellshape",
+                "mergeable": true,
+                "option": "spellshape",
+                "placement": "spellcasting",
+                "predicate": [
+                    "feature:arcane-bond"
+                ],
+                "suboptions": [
+                    {
+                        "label": "{item|name}",
+                        "value": "bond-conservation"
+                    }
+                ],
+                "toggleable": true,
+                "value": true
```

Comment:
This value in your Roll Options snuck from your testing most likely, it should be removed from them